Makefile

```
@@ -400,7 +400,7 @@ proto-check-breaking:
 	@$(DOCKER_BUF) breaking --against $(PR_TARGET_REPO)#branch=$(PR_TARGET_BRANCH)
 
 
-TM_URL              = https://raw.githubusercontent.com/tendermint/tendermint/v0.34.0-rc6/proto/tendermint
+TM_URL              = https://raw.githubusercontent.com/tendermint/tendermint/v0.34.22/proto/tendermint
```

Should point to our fork?

```suggestion
TM_URL              = https://raw.githubusercontent.com/agoric-labs/tendermint/v0.34.23-alpha.agoric.2/proto/tendermint
```